<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Open_job</class>
 <widget class="QDialog" name="Open_job">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ouvrir un job CarHab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>438</width>
        <height>85</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lbl_open_job">
         <property name="text">
          <string>Job (.sqlite) : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_edit_open_job"/>
       </item>
       <item>
        <widget class="QPushButton" name="psh_btn_open_job">
         <property name="text">
          <string>Parcourir...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btn_box_open_job">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_box_open_job</sender>
   <signal>accepted()</signal>
   <receiver>Open_job</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_box_open_job</sender>
   <signal>rejected()</signal>
   <receiver>Open_job</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
